### PR TITLE
panther aws pack

### PR DESCRIPTION
**Pull request by lindsey-w**
### Background

Asana URL: Closes https://app.asana.com/0/1199940726973121/1199936510096869/f

Interest in creating a panther-managed "aws" pack that consists of the most critical, high value detections that can be used out of the box.

### Changes

* created core panther aws pack

### Testing

* `make ci`

image

image

### Additional Details
- Included _all_ High & Critical Sev Policies & Rules
    - **_Except_**: those with "Configuration Required" Tags
- Included the following Medium Severity Rules:
    - AWS.Password.Unused
    - AWS.PasswordPolicy.PasswordAgeLimit
    - AWS.CloudTrail.AMIModifiedForPublicAccess
    - AWS.Config.RecordingNoErrors
        - nice to combine with High Sev `AWS.Config.RecordingEnabled` detection
    - Standard.BruteForceByIP
- Excluded:
    - Anything already auto-disabled out of the box
    - All INFO level severities
        - **_Note_**: in the future, it would be cool to define an "informational aws pack" that has a pack definition like... `LogType: aws.* ResourceType: aws.* Severity: INFO` 
            - this would be especially nice when combine with the auto-resolve INFO level alerts feature
    - AWS.NetworkACL.RestrictedSHH
        - SSH isn't guaranteed to be running, so perhaps don't default enable this
    - AWS.RootAccount.HardwareMFA
        - MFA for root account is monitored via `AWS.RootAccount.MFA` this is just icing on the cake

### CODE REVIEW

**Review by nhakmiller — APPROVED** [review]
LGTM. We may need to tune in the networking policies a bit if we're pushing them out to the world, but luckily with packs we can do that after that fact! Woohoo!

File: packs/aws.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    - AWS.EC2.Volume.Encryption
+    - AWS.EC2.Volume.Snapshot.Encrypted
+    - AWS.Redshift.Cluster.Encryption
+    - AWS.RDS.Instance.Encryption
+    - AWS.S3.Bucket.Encryption
+    # Networking Policies
+    - AWS.NetworkACL.RestrictsInboundTraffic
+    - AWS.SecurityGroup.AdministrativeIngress
+    - AWS.SecurityGroup.OnlyDMZPubliclyAccessible
+    - AWS.SecurityGroup.RestrictsInboundTraffic
```
Comment (wey-chiang) [inline packs/aws.yml:29]:
```suggestion
    # Networking Policies
    - AWS.NetworkACL.RestrictsInboundTraffic
    - AWS.SecurityGroup.AdministrativeIngress
    - AWS.SecurityGroup.OnlyDMZPubliclyAccessible
    - AWS.SecurityGroup.RestrictsAccessToCDE
    - AWS.SecurityGroup.RestrictsInboundTraffic
```

**Review by wey-chiang — COMMENTED** [review]
This policy is default disabled but might be worth including as it is marked `Critical`

**Review by wey-chiang — APPROVED** [review]
Did some basic grep on `panther-analysis` and comparison for the criteria you listed, LGTM 👍

File: packs/aws.yml (hunk 1 above)
Comment (lindsey-w) [reply packs/aws.yml:29]: Nice find! I was considering it, but the policy body mentions:

```
# This is a generic policy that checks inbound permissions on a Security Group.
# You may wish to add additional logic specific to your use cases.
```

which to me seems a bit between "configuration required" and not needing configuration. I'm on the fence about it - so if you have a strong opinion that it should be included I can do that :)

**Pull request merged**